### ISSUE TITLE
Parallel downloads of MD5SUM files?

### ISSUE THREAD

**Issue by kblin** [kblin/ncbi-genome-download#47]
`ncbi-genome-download` can download data in parallel already to speed up the process. But the part of the process that downloads the MD5SUMS files and checks if a download job needs to be started in the first place is still one at a time. Fix this to be parallel already.

**Comment by kblin** [comment]
This will be a major structure change internally, so not planned any time soon. Certainly nice to have.

**Comment by kblin** [comment]
Closed by #103, thanks @axbazin !
